src/Storage/Device/S3.php

```diff
@@ -915,11 +941,20 @@ protected function call(string $method, string $uri, string $data = '', array $p
 
         $result = \curl_exec($curl);
 
+        $response->code = \curl_getinfo($curl, CURLINFO_HTTP_CODE);
+
+        $attempt = 0;
+        while ($attempt < self::$retryAttempts && $response->code === 503) {
+            usleep(self::$retryDelay);
```

Comment:
`usleep` use microseconds instead of millis; this will only sleep 0.5 milliseconds

Reply:
updated